test/check-machines-networks

```diff
@@ -786,6 +786,50 @@ class TestMachinesNetworks(VirtualMachinesCase):
         b.click(".pf-c-modal-box__footer button:contains(Delete)")
         self.waitNetworkRow("test_network2", connectionName, False)
 
+    def testNetworkAddStaticDCHPHosts(self):
+        b = self.browser
+        m = self.machine
+
+        self.createVm("subVmTest1", running=False)
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual machines")
+
+        # Click on Networks card
+        b.click(".pf-c-card .pf-c-card__header button:contains(Network)")
+
+        # Check that all networks are there
+        b.wait_in_text("body", "Networks")
+        self.waitNetworkRow("default")
+
+        # Expand row for first network
+        self.toggleNetworkRow("default")
+
+        # Check that adding new entries is available only for inactive networks
+        b.wait_visible("#network-default-system-static-host-entries-add:disabled")
+
+        m.execute("virsh net-destroy default")
+
+        # Set up a static IP address
+        mac = self.getDomainMacAddress("subVmTest1")
+        b.click("#network-default-system-static-host-entries-add")
+        b.set_input_text("#add-new-static-entry-mac-address", mac)
+        b.set_input_text("#add-new-static-entry-ip-address", "192.168.122.222")
+        b.click("#add-new-static-entry-save")
+        b.wait_in_text("#network-default-system-ipv4-dhcp-host-0", f"MAC: {mac}, IP: 192.168.122.222")
+
+        # Try to set up the same MAC -> IP mapping and expect error
+        b.click("#network-default-system-static-host-entries-add")
+        b.set_input_text("#add-new-static-entry-mac-address", self.getDomainMacAddress("subVmTest1"))
+        b.set_input_text("#add-new-static-entry-ip-address", "192.168.122.222")
+        b.click("#add-new-static-entry-save")
+        b.wait_in_text(".pf-c-modal-box .pf-c-alert.pf-m-danger", "there is an existing dhcp host entry")
+        b.click("#add-new-static-entry-cancel")
+
+        # Check that the static IP is correctly assigned to the VM
+        m.execute("virsh net-start default && virsh start subVmTest1")
+        wait(lambda: "1" in self.machine.execute("virsh domifaddr subVmTest1 | grep 192.168.122.222 | wc -l"), delay=3)
+
```

Comment:
We should also test the same for ipv6, as that has slightly different XML format for <host> section

Reply:
I think I will remove the ipv5 host entries adding from this PR.